devices/tuya.js

```diff
@@ -626,6 +626,45 @@ module.exports = [
             }
         },
     },
+    {
+        fingerprint: [{modelID: 'TS011F', manufacturerName: '_TZ3000_cphmq0q7'},],
+        model: 'TS011F',
+        description: 'Smart EU Plug',
+        vendor: 'TuYa',
+        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering, fz.ignore_basic_report, fz.tuya_switch_power_outage_memory],
```

Comment:
Do you get an updated `state` when controlling the plug via the button on it?

Reply:
Yes

Reply:
perfect